metalearn/metafeatures/general_resource_computers.py

```diff
@@ -0,0 +1,247 @@
+import numpy as np
+import pandas as pd
+from sklearn.model_selection import StratifiedShuffleSplit
+
+from metalearn.metafeatures.base import ResourceComputer
+import metalearn.metafeatures.constants as consts
+
```

Comment:
The functions were all factored out of the `Metafeatures` class, so that class could be smaller and so they could be decorated with `ResourceComputer`.